modules/5_install/install.tf

```diff
@@ -133,6 +133,11 @@ locals {
 }
 
 resource "null_resource" "config" {
+
+    triggers = {
+       worker_count = length(var.worker_ips)
```

Comment:
Can you please explain this trigger ?

Reply:
Whenever there is a change in the trigger attributes the resource would be re-created after apply. Here whenever the count changes it will run the helpernode again. This is needed for changing the configuration for named, dhcp, etc.

Reply:
@yussufsh we are going to need this on kvm also.   Is that on your list?

Reply:
I do not have a KVM box to implement this at this time. I am working on getting my test setup.
Only issue I am worried about is the use of module dependency we have used in kvm repo. Once I have a test setup I will be able to contribute this change there.